File: PracticasProgramacion/Practica_24/python/main.py
"""
Practica_24 - Escribir y leer CSV (datos basicos, arreglos, autos, personas ADT).
"""

from manejador_csv import ManejadorCSV

if __name__ == "__main__":
    man = ManejadorCSV()
    man.escribir_datos_basicos("datos_basicos.csv")
    man.escribir_arreglo("arreglo.csv")
    man.escribir_autos("autos.csv")
    man.escribir_personas("personas.csv")

    print("Leyendo datos basicos:")
    man.leer_y_mostrar("datos_basicos.csv")
    print("\nLeyendo arreglo:")
    man.leer_y_mostrar("arreglo.csv")
    print("\nLeyendo autos:")
    man.leer_y_mostrar("autos.csv")
    print("\nLeyendo personas (ADT):")
    man.leer_y_mostrar("personas.csv")

File: PracticasProgramacion/Practica_24/cpp/main.cpp
/**
 * Practica_24 - Escribir y leer CSV (datos basicos, arreglos, creados, ADT)
 */

#include "ManejadorCSV.h"
#include <iostream>

int main() {
    ManejadorCSV man;

    man.escribirDatosBasicos("datos_basicos.csv");
    man.escribirArreglo("arreglo.csv");
    man.escribirAutos("autos.csv");
    man.escribirPersonas("personas.csv");

    std::cout << "Leyendo datos basicos:\n";
    man.leerYMostrar("datos_basicos.csv");
    std::cout << "\nLeyendo arreglo:\n";
    man.leerYMostrar("arreglo.csv");
    std::cout << "\nLeyendo autos:\n";
    man.leerYMostrar("autos.csv");
    std::cout << "\nLeyendo personas (ADT):\n";
    man.leerYMostrar("personas.csv");

    return 0;
}
